src/www/firewall_rules_edit.php

```diff
@@ -1376,10 +1376,10 @@ function is_posnumericint($arg) {
                               <?=gettext("Hint: Select which type of state tracking mechanism you would like to use. If in doubt, use keep state.");?>
                             </span>
                               <ul>
-                                <li><strong><?=gettext("keep state");?> </strong><?=gettext("Works with all IP protocols.");?></li>
-                                <li><strong><?=gettext("sloppy state");?> </strong><?=gettext("Works with all IP protocols.");?></li>
-                                <li><strong><?=gettext("synproxy state");?> </strong><?=gettext("Proxies incoming TCP connections to help protect servers from spoofed TCP SYN floods. This option includes the functionality of keep state and modulate state combined.");?></li>
-                                <li><strong><?=gettext("none");?> </strong><?=gettext("Do not use state mechanisms to keep track.  This is only useful if you're doing advanced queueing in certain situations.  Please check the documentation.");?> </li>
+                                <li><?= sprintf(gettext('%sKeep state%s works with all IP protocols.'),'<strong>', '</strong>') ?></li>
+                                <li><?= sprintf(gettext('%sSloppy state%s works with all IP protocols.'),'<strong>', '</strong>') ?></li>
+                                <li><?= sprintf(gettext('%sSynproxy state%s proxies incoming TCP connections to help protect servers from spoofed TCP SYN floods. This option includes the functionality of keep state and modulate state combined.'),'<strong>', '</strong>') ?></li>
+                                <li><?= sprintf(gettext("%sNone%s: Do not use state mechanisms to keep track. This is only useful if you're doing advanced queueing in certain situations. Please check the documentation."),'<strong>', '</strong>') ?></li>
```

Comment:
Honestly, not sure what to do here. This is a case of excessive documentation, it would be better to simply point to the pf.conf(5) man page and advertise the states as is or at least with a brief translation in the actual option selection?